### PR TITLE
Feature/consumption.point

### PR DESCRIPTION
**Pull request by oriolcanades**
- NEW data model of ConsumptionPoint
- NEW data model of ConsumptionCost

### CODE REVIEW

File: CROSSSECTOR/Consumption.Point/dataModel.ConsumptionCost/ADOPTERS.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
-description: This is a compilation list of the current adopters of the data model [Data model] of the Subject [Subject].  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
-currentAdopters:
--
- adopter: 
- description: 
- mail: 
- organization: 
- project: 
- comments: 
+description: This is a compilation list of the current adopters of the data model [Data model] of the Subject [Subject].  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```
Comment (albertoabellagarcia) [inline CROSSSECTOR/Consumption.Point/dataModel.ConsumptionCost/ADOPTERS.yaml:1]: please fill the fields and remove the templated content

File: CROSSSECTOR/Consumption.Point/dataModel.ConsumptionCost/notes.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
-notesHeader:
-  notes appearing at the beginning of the spec
-notesMiddle:
-  notes appearing in the middle of the spec
-notesFooter:
-  notes appearing in the footer of the spec
-notesReadme:
+notesHeader:
```
Comment (albertoabellagarcia) [inline CROSSSECTOR/Consumption.Point/dataModel.ConsumptionCost/notes.yaml:1]: please remove templated contents

File: CROSSSECTOR/Consumption.Point/dataModel.ConsumptionPoint/ADOPTERS.yaml [hunk 3]
```diff
@@ -1,10 +1,10 @@
-description: This is a compilation list of the current adopters of the data model [Data model] of the Subject [Subject].  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
-currentAdopters:
--
- adopter: 
- description: 
- mail: 
- organization: 
- project: 
- comments: 
+description: This is a compilation list of the current adopters of the data model [Data model] of the Subject [Subject].  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```
Comment (albertoabellagarcia) [inline CROSSSECTOR/Consumption.Point/dataModel.ConsumptionPoint/ADOPTERS.yaml:1]: remove templated contents and use the proper name for the data model.

File: CROSSSECTOR/Consumption.Point/dataModel.ConsumptionPoint/notes.yaml [hunk 4]
```diff
@@ -1,8 +1,8 @@
-notesHeader:
-  notes appearing at the beginning of the spec
-notesMiddle:
-  notes appearing in the middle of the spec
-notesFooter:
-  notes appearing in the footer of the spec
-notesReadme:
+notesHeader:
```
Comment (albertoabellagarcia) [inline CROSSSECTOR/Consumption.Point/dataModel.ConsumptionPoint/notes.yaml:1]: remove the templated content

**Review by albertoabellagarcia — COMMENTED** [review]
please attend the minimum comments on these files

File: CROSSSECTOR/Consumption.Point/dataModel.ConsumptionCost/ADOPTERS.yaml (hunk 1 above)
Comment (oriolcanades) [reply CROSSSECTOR/Consumption.Point/dataModel.ConsumptionCost/ADOPTERS.yaml:1]: Added new data for ADOPTERS.yaml

File: CROSSSECTOR/Consumption.Point/dataModel.ConsumptionCost/notes.yaml (hunk 2 above)
Comment (ocanades) [reply CROSSSECTOR/Consumption.Point/dataModel.ConsumptionCost/notes.yaml:1]: Added new data for ADOPTERS.yaml

File: CROSSSECTOR/Consumption.Point/dataModel.ConsumptionPoint/ADOPTERS.yaml (hunk 3 above)
Comment (oriolcanades) [reply CROSSSECTOR/Consumption.Point/dataModel.ConsumptionPoint/ADOPTERS.yaml:1]: Added new data for ADOPTERS.yaml

File: CROSSSECTOR/Consumption.Point/dataModel.ConsumptionCost/notes.yaml (hunk 2 above)
Comment (oriolcanades) [reply CROSSSECTOR/Consumption.Point/dataModel.ConsumptionCost/notes.yaml:1]: deleted templated contets

File: CROSSSECTOR/Consumption.Point/dataModel.ConsumptionPoint/notes.yaml (hunk 4 above)
Comment (oriolcanades) [reply CROSSSECTOR/Consumption.Point/dataModel.ConsumptionPoint/notes.yaml:1]: deleted templated contets

File: CROSSSECTOR/Consumption.Point/dataModel.ConsumptionPoint/ADOPTERS.yaml (hunk 3 above)
Comment (albertoabellagarcia) [reply CROSSSECTOR/Consumption.Point/dataModel.ConsumptionPoint/ADOPTERS.yaml:1]: I'll fix it the minor issues here

**Review by albertoabellagarcia — APPROVED** [review]

**Pull request merged**